react/features/prejoin/actionTypes.js

```diff
@@ -14,6 +14,11 @@ export const SET_DEVICE_STATUS = 'SET_DEVICE_STATUS';
  */
 export const SET_SKIP_PREJOIN = 'SET_SKIP_PREJOIN';
 
+/**
+ * Action type used to set the mandatory stance of the prejoin display name.
+ */
+export const SET_DISPLAY_NAME_REQUIRED = 'SET_DISPLAY_NAME_REQUIRED';
```

Comment:
Can we please change this to something with "prejoin" in the name? Like `SET_PREJOIN_DISPLAY_NAME_REQUIRED`.

We have another config option for requiring display names, so it might get confusing.

Reply:
Also, please adapt the action name to follow suit.